pkg/util/podvolume/pod_volume.go

```diff
@@ -140,6 +318,9 @@ func getPodVolumeNameForPVC(pod corev1api.Pod, pvcName string) (string, error) {
 	return "", errors.Errorf("Pod %s/%s does not use PVC %s/%s", pod.Namespace, pod.Name, pod.Namespace, pvcName)
 }
 
+// GetPodsUsingPVC returns all pods in the given namespace that use the specified PVC.
+// This function lists all pods in the namespace and filters them, which can be slow
+// when there are many pods. Consider using GetPodsUsingPVCWithCache for better performance.
```

Comment:
If this one is not used by plugins and it is deprecated, why don't we remove it directly?